test/functional/zerocoin_publicSpend_reorg.py

```diff
@@ -0,0 +1,208 @@
+# !/usr/bin/env python3
```

Comment:
remove the whitespace between the `#` and `!` here. File also needs to be made executable